app/controllers/invites_controller.rb

```diff
@@ -100,7 +107,7 @@ def create
     end
   end
 
-  def create_invite_link
+  def create_single_use_invite_link
```

Comment:
this to me API wise is a bit confusing, why do we need a special end point for "single vs multiple" 

Isn't "single" simply an invite link with max_redemptions_allowed == 1 ?

Reply:
You're correct. I think I can replace these two methods with a single method `create_invite_link` wherein single use and multiple use can be differentiated via a param.

Reply:
Done.